back/app/models/custom_field.rb

```diff
@@ -261,6 +263,20 @@ def custom_form_type?
     resource_type == 'CustomForm'
   end
 
+  def user_type?
+    resource_type == 'User'
+  end
+
+  def items_claz
```

Comment:
Minor comment nothing to do with this PR - sometimes we use `claz` and sometimes we use `klass` in the codebase. Might be nice to standardise

Reply:
:+1: I'll see what's most common and change accordingly.

Reply:
![image](https://github.com/user-attachments/assets/fe9faf98-c518-41b9-8988-77e3b2c14cb9)

![image](https://github.com/user-attachments/assets/7e89437f-e492-4f2a-ad51-b7e4e66af7a4)

Pretty close, so I'll just leave it